katacomb/scripts/uv_export.py

```diff
@@ -7,7 +7,7 @@
 import katdal
 
 import katacomb.configuration as kc
-from katacomb import pipeline_factory, AIPSPath
+from katacomb import pipeline_factory, AIPSPath, fits_dir
```

Comment:
Do you need the `fits_dir` import here ?

Reply:
Yup. Not needed. Thanks.